docs/user_guide/1_4_simulated_hand_mujoco.rst

```diff
@@ -0,0 +1,88 @@
+Mujoco
+=======
+
+`Mujoco <http://www.mujoco.org/>`_ is a robot simulator that has now been adopted by a wide community of researchers and developers, specially for
+machine learning applications. We have developed the tools and the model of our dexterous hand to use Mujoco as an alternative to Gazebo. 
+Mujoco is not free so follow the next instructions if you have already a `Mujoco License <https://www.roboti.us/license.html>`_.
+
+
+Obtaining the mujoco simulation
```

Comment:
`mujoco` should start with a capital letter as in other places.

Reply:
done